src/radical/pilot/session.py

```diff
@@ -295,8 +295,8 @@ def close(self, **kwargs):
 
             self._prof.prof("session_fetch_start", uid=self._uid)
             self._log.debug('start download')
-            tgt = os.getcwd()
-            self.fetch_json    (tgt='%s/%s' % (tgt, self.uid))
+            tgt = self._cfg.base
+            self.fetch_json    (tgt='%s/%s' % (self._cfg.path, self.uid))
```

Comment:
should both changes be `self._cfg.base`? (`path` contains `base`)
(*)
```python
            if not self._cfg.path:
                self._cfg.path = '%s/%s' % (self._cfg.base, self._cfg.sid)
```

Reply:
Yes, otherwise we end up with `base/sid/sid` like path's.  Thanks for checking though, I got that very thing wrong on the first fix attempt :-)